driver-mapping/src/main/java/com/datastax/driver/mapping/AnnotationParser.java

```diff
@@ -194,6 +194,9 @@ else if (propertyMapper.isClusteringColumn())
                 throw new IllegalArgumentException(String.format("Field %s does not exist in type %s.%s",
                         propertyMapper.columnName, ksName, userType.getTypeName()));
 
+            for (Class<?> fieldUdt : TypeMappings.findUDTs(field.getGenericType()))
```

Comment:
Your fix is in the right place, however, you need to adjust the code to:

```
for (Class<?> fieldUdt : TypeMappings.findUDTs(propertyMapper.javaType.getType()))
                mappingManager.getUDTCodec(fieldUdt);
```

Because some properties might only be present through getters and setters and not through fields.